system-tests/tests/load/cre/workflow-don-load-test-streams-kind.toml

```diff
@@ -0,0 +1,172 @@
+duration = "2h" # specific to chaos test, ignore this value if you are performing a chaos test
+
+# ----- BLOCKCHAIN CONFIGURATION -----
+# Sets up a local Anvil blockchain for testing with 5-second block time
+[[blockchains]]
+chain_id = "1337"
+type = "anvil"
+docker_cmd_params = ["-b", "5"]
+
+# ----- BINARY PATHS -----
+# Path to the mock capability binary used for test simulation
+[binaries_config]
+mock_capability_binary_path = "../../../../../capabilities/bin/amd64/mock"
+
+# ----- JOB DISTRIBUTOR -----
+# Docker image for the job distributor service
+[jd]
+image = "localhost:5000/job-distributor:sha-707362c-amd64"
+
+# ----- MOCK CAPABILITIES -----
+# Trigger capability that simulates streams of data
+[[mock_capabilities]]
+name="streams-trigger"
+version= "2.0.0"
+type="trigger"
+description="streams trigger mock"
+
+# Target capability that simulates writing data to Ethereum
+[[mock_capabilities]]
+name="write_ethereum"
+version= "1.0.0"
+type="target"
+description="write trigger mock"
+
+# ----- TEST CONFIGURATION -----
+# Parameters for the Workflow DON load test
+[workflow_load]
+streams = 1000              # Number of streams to simulate
+jobs = 2                    # Number of jobs per stream
+
+# ----- INFRASTRUCTURE SETUP -----
+# Configures Docker-based test infrastructure
+[infra]
+type = "docker"
+
+# CRIB deployment configuration for local KIND cluster
+[infra.crib]
+namespace = "crib-workflow-don-load-test"
+folder_location = "../../../../../crib/deployments/cre"
```

Comment:
maybe all paths like this one or capability binary paths should be replaced with placeholders + meaningful comment regarding expected value?